airbyte-webapp/src/components/connectorBuilder/types.ts

```diff
@@ -45,36 +49,40 @@ export const builderFormValidationSchema = yup.object().shape({
 export const convertToManifest = (values: BuilderFormValues): ConnectorManifest => {
   const manifestStreams: DeclarativeStream[] = values.streams.map((stream) => {
     return {
+      type: "DeclarativeStream",
       name: stream.name,
       retriever: {
+        type: "SimpleRetriever",
         name: stream.name,
         requester: {
+          type: "HttpRequester",
           name: stream.name,
           url_base: values.global?.urlBase,
           path: stream.urlPath,
           request_options_provider: {
+            // TODO can't declar type here because the server will error out, but the types dictate it is needed. Fix here once server is fixed.
```

Comment:
```suggestion
            // TODO can't declare type here because the server will error out, but the types dictate it is needed. Fix here once server is fixed.
```